Example/SmileID/Jobs/JobItemModel.swift

```diff
@@ -58,6 +60,8 @@ class JobItemModel: ObservableObject {
     @MainActor
     func updateJobStatus() async throws {
         guard !job.jobComplete, task == nil else { return }
+        isLoading = true
+        defer { isLoading = false }
```

Comment:
I didn't know about this, nice!

Reply:
It's commonly used in a scenario where you're not sure when the block of code will end and that makes sure to run the statement after everything else.